tools/deploy_partition.py

```diff
@@ -159,6 +160,8 @@ def sign_firmware(data: bytes, priv_key: Any) -> bytes:
 
 def main(args):
   colorama.init()
+  if not args.priv_key:
```

Comment:
This test is already done a few lines later, making it redundant.

Reply:
Unintended artifact from rebasing.